main/src/library/Option.flix

```diff
@@ -174,6 +174,14 @@ mod Option {
     ///
     pub def nonEmpty(o: Option[a]): Bool = not isEmpty(o)
 
+    ///
+    /// Returns `1` for `Some(_)` and `0` for `None`.
+    ///
+    pub def size(o: Option[a]): Int32 = match o {
+        case Some(_) => 1
+        case None => 0
```

Comment:
Swap order

Reply:
Also: Why? Remember we dont have isNone or isSome. So this should also not be here.

Reply:
I think its nice, but not enough to champion it 🤷‍♂️